autopark/VehicleType.cs

```diff
@@ -11,10 +11,10 @@ class VehicleType
         public string TypeName { get; set; }
         public float TaxCoefficient { get; set; }       
 
-        public VehicleType(string typeName, float taxCoefficient)
+        public VehicleType(string typeName)
         {
             this.TypeName = typeName;
-            this.TaxCoefficient = taxCoefficient;
+            this.TaxCoefficient = GetTaxCoefficient(typeName);
```

Comment:
Return old version of VehicleType.

Reply:
![image](https://user-images.githubusercontent.com/79978383/115968588-49876780-a541-11eb-8933-587d39d938cd.png)
Done